### ISSUE TITLE
Show file size for all files

### ISSUE THREAD

**Issue by AlexIII** [mkxml/vscode-filesize#32]
The extension only shows size of the file if it's open in a tab. 
It is would be very useful if it also showed size of any binary file that not necessary can be opened via vscode.

**Comment by Siddhantshukla814** [comment]
Yeah! Please also add support for all image formats eg: jpeg, png, svg etc.
